help/destinations/catalog/mobile-engagement/moengage.md

```diff
@@ -0,0 +1,142 @@
+---
+keywords: mobile;mobile engagement destinations;moengage;moengage mobile engagement destination
+title: Moengage connection
+description: [!DNL Moengage]  is a customer engagement platform that powers customer-centric interactions between consumers and brands in real time.
+---
+
+# [!DNL Moengage] connection
+
+## Overview {#overview}
+
+[!DNL Moengage] destination allows you to connect and map your Adobe data (user attributes, segments and events) to MoEngage in real-time. Customers can then act on this data, delivering personalized, targeted experience.
```

Comment:
Really minor, and just a suggestion, but you could reshuffle the wording a bit to really address the reader, like: 

Use the [!DNL Moengage] destination to connect and map your Adobe data (user attributes, segments and events) to MoEngage in real-time.

Reply:
done